quodlibet/po/fr.po

```diff
@@ -2652,18 +2644,16 @@ msgid ""
 msgstr ""
 
 #: ../quodlibet/ext/gstreamer/karaoke.py:20
-#, fuzzy
 msgid "Filter _band:"
-msgstr "Filtrer sur ce _genre"
+msgstr "Filtrer sur le groupe"
```

Comment:
Here it's band as in frequency band, so "Bande de fréquence du filtre" or "Bande du filtre" or "Bande"